Assets/Scripts/ViconNexusUnityStream/CustomHWDScript.cs

```diff
@@ -29,6 +29,7 @@ public class CustomHWDScript : CustomSubjectScript
 
         private OneEuroFilter<Quaternion> rotFilter;
         private OneEuroFilter<Vector3> posFilter;
+        [SerializeField] private float HMDCentreOffset;
```

Comment:
This a better place to have this than in the project settings. Consider changing this to a Vector3, then remove the `HMDPositionOffset` and it's references across?